src/main/java/customerrelationsmanagement/Tables.java

```diff
@@ -0,0 +1,140 @@
+package customerrelationsmanagement;
+
+import java.util.InputMismatchException;
+
+public enum Tables {
```

Comment:
Love to see the stuff we've learned in class to be added into our projects. Creating the tables to be enum is a fantastic way to have a 'solid' object. Considering we should've done this way earlier for our tables, I'm glad to at least exists in our final product review